### PR TITLE
Remove draft/live feature flag

### PR DESCRIPTION
**Pull request by jamie-o-wilkinson**
#### What problem does the pull request solve?
We have launched “Draft/live” feature a good number of weeks now and we have no need to toggle it on/off. It also helps simplify the codebase and reduces cognitative load of trying to understand how our service works.

Trello card:
https://trello.com/c/F95ojogB

#### Checklist

- [x] I've used the pull request template
- [x] I've linked this PR to the relevant issue (if mission work)
- [x] I've written unit tests for these changes (if code change)
- N/A I've updated the documentation in (If any documentation requires updating)
  - N/A README.md
  - N/A Elsewhere (please link)

### CODE REVIEW

File: app/components/task_list_component/view.rb [hunk 1]
```diff
@@ -14,9 +14,8 @@ def initialize(completed_task_count: nil, total_task_count: nil, sections: [], c
 
     def render_counter?
       return false unless @completed_task_count.present? && @total_task_count.present?
-      return true if FeatureService.enabled?(:draft_live_versioning)
 
-      @completed_task_count != @total_task_count
+      true
```
Comment (lfdebrux) [inline app/components/task_list_component/view.rb:18]: The logic of this is a bit hard to follow, can we simplify it to just

```suggestion
      @completed_task_count.present? && @total_task_count.present?
```

File: spec/components/task_list_component/view_spec.rb [hunk 2]
```diff
@@ -110,42 +110,20 @@
...
-                rows: [
-                  { task_name: "task a", path: "#", status:, active: },
-                ] },
-              { title: "section 2",
-                rows: [
-                  { task_name: "task d", path: "#", status:, active: },
-                ] },
-            ]))
-          end
-
```
Comment (lfdebrux) [inline spec/components/task_list_component/view_spec.rb:130]: Nice 👍

**Review by lfdebrux — APPROVED** [review]
Awesome stuff, thanks! Just one comment, otherwise looks good.

File: spec/config/settings_spec.rb [hunk 3]
```diff
@@ -21,7 +21,7 @@
   describe ".features" do
     features = settings[:features]
 
-    include_examples expected_value_test, :draft_live_versioning, features, false
+    include_examples expected_value_test, :basic_routing, features, false
```
Comment (jamie-o-wilkinson) [inline spec/config/settings_spec.rb:24]: Noticed this was missing so replaced `draft_live_versioning` with `basic_routing`
